### PR TITLE
DRA: Fix ResourceClaimTemplateSpec comment

### PR DESCRIPTION
**Pull request by LionelJouin**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Fix comment in the ResourceClaimTemplateSpec (resource.k8s.io/v1alpha3).
ObjectMeta for the ResourceClaimTemplateSpec are copied into ResourceClaim not PVC.

#### Which issue(s) this PR fixes:

/

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/api/resource/v1alpha3/types.go [hunk 1]
```diff
@@ -1021,7 +1021,7 @@ type ResourceClaimTemplate struct {
 
 // ResourceClaimTemplateSpec contains the metadata and fields for a ResourceClaim.
 type ResourceClaimTemplateSpec struct {
-	// ObjectMeta may contain labels and annotations that will be copied into the PVC
+	// ObjectMeta may contain labels and annotations that will be copied into the ResourceClaim
```
Comment (pohly) [inline staging/src/k8s.io/api/resource/v1alpha3/types.go:1024]: Thanks. Please also fix in `pkg/apis/resource/types.go`, then we can try to get this approved.

Comment (LionelJouin) [reply staging/src/k8s.io/api/resource/v1alpha3/types.go:1024]: Done

**Review by pohly — APPROVED** [review]
/lgtm
/assign @thockin 

For approval of a copy-and-paste error in the DRA API.

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**